packages/components/src/internal/components/forms/QueryFormInputs.tsx

```diff
@@ -249,6 +251,7 @@ export class QueryFormInputs extends React.Component<QueryFormInputsProps, State
                                         joinValues={joinValues}
                                         label={col.caption}
                                         addLabelAsterisk={showAsteriskSymbol}
+                                        renderFieldLabel={renderFieldLabel}
```

Comment:
Really hope we can use this in more places to make the labels actually useful to users.

Reply:
Agreed